kpops/cli/main.py

```diff
@@ -205,6 +205,17 @@ def generate(
         None, help="Path to file the yaml pipeline should be saved to"
     ),
     verbose: bool = typer.Option(False, help="Enable verbose printing"),
+    template: bool = typer.Option(False, help="Run helm template"),
+    steps: Optional[str] = PIPELINE_STEPS,
+    api_versions: str = typer.Option(
+        "", help="Kubernetes api version used for Capabilities.APIVersions"
+    ),
+    ca_file: str = typer.Option(
```

Comment:
why not `str | None` with a default `None`?

Reply:
I don't know, tbh. What are the advantages of keeping the strings at a default None?

Reply:
main.py

Reply:
please use `Optional[str]` with a default `None` for Typer, as it doesn't support the new syntax

Reply:
done

Reply:
you forgot to change `str` to `Optional[str]`

Reply:
fixed it now, sorry